tools/languagelocalizer/languageLocalizer.js

```diff
@@ -128,36 +38,41 @@ const LanguageLocalizer = {
 
   async init() {
     program.version('0.0.1')
+    program.option("-c, --check", `
+      Check only, will fail if user input would be required to generate 
+      the file.
+    `);
 
-    program.option(
```

Comment:
How come this was removed?

Reply:
That was used to set where the `mozillavpn.exe` is that will be started to get it's languages. 
With this change we will instead check the `i18n` repo, so there is no need to know where the binary is :)